Reports/router.js

```diff
@@ -1,4 +1,38 @@
 var express = require('express');
+var mongoose = require('mongoose');
+mongoose.connect('mongodb://localhost:27017');
```

Comment:
Instead of calling ```mongoose.connect('mongodb://localhost:27017')``` in every file that uses mongoose, we can call ```mongoose.connect(...)``` once in server.js, and all references to mongoose will already be connected (i.e., no other files would need to call ```mongoose.connect(...)```).  That way, we wouldn't need a global variable or ```.config``` entry for URL:PORT.

We could still keep this information in a ```.config``` or global variable, but I don't think we'd need to in order to conform to SSOT.